apisix/plugins/limit-count.lua

```diff
@@ -171,7 +170,29 @@ function _M.access(conf, ctx)
         return 500
     end
 
-    local key = (ctx.var[conf.key] or "") .. ctx.conf_type .. ctx.conf_version
+    local conf_key = conf.key
+    local key
+    if conf.key_type == "var_combination" then
+        local err, n_resolved
+        key, err, n_resolved = core.utils.resolve_var(conf_key, ctx.var);
+        if err then
+            core.log.error("could not resolve vars in ", conf_key, " error: ", err)
+        end
+
+        if n_resolved == 0 then
+            key = nil
+        end
+    else
+        key = ctx.var[conf_key]
```

Comment:
CI failure occurred, I wonder if I should use 'key = ( ctx.var[conf_key] or "")' instead of the current way. cc @spacewander

Reply:
if `ctx.var[conf_key]` is `nil`, what is the result you expect?

Reply:
Yes, I got it, thanks~